beaker-tests/Sanity/copr-cli-basic-operations/runtest-modules.sh

```diff
@@ -142,7 +142,7 @@ rlJournalStart
         # $CHROOT variable
         rlRun "copr-cli create $PROJECT --chroot $PREV_CHROOT --chroot $CHROOT --chroot fedora-rawhide-i386"
         # move back to modules/testmodule once this is merged https://src.fedoraproject.org/modules/testmodule/pull-request/1
-        rlRun "copr-cli build-module --distgit fedora --url https://src.fedoraproject.org/fork/praiskup/modules/testmodule/raw/fix-rawhide/f/testmodule.yaml $PROJECT"
+        rlRun "copr-cli build-module --distgit fedora --url https://src.fedoraproject.org/fork/nikromen/modules/testmodule/raw/fix-rawhide/f/testmodule.yaml $PROJECT"
```

Comment:
I'd have given you permissions to that fork but np.

Reply:
I was doing that at friday's night - I didn't want to bother you that late